src/main/java/org/intellij/sdk/codesync/CodeSyncLogger.java

```diff
@@ -15,6 +16,7 @@
 import static org.intellij.sdk.codesync.Constants.*;
 
 import java.io.FileNotFoundException;
+import java.net.UnknownHostException;
```

Comment:
@syedgulahmed are we using it somewhere?

Reply:
No, I will remove it.

Reply:
cool, thanks